tools/data/super-resolution/reds/preprocess_reds_dataset.py

```diff
@@ -201,7 +201,7 @@ def parse_args():
 
 if __name__ == '__main__':
     """You can first download datasets using the scripts:
-    https://gist.github.com/SeungjunNah/b10d369b92840cb8dd2118dd4f41d643
+    https://gist.github.com/SeungjunNah/b10d369b92840cb8dd2118dd4f41d643.
```

Comment:
Should we append a dot here?

Reply:
It's appended here automatically by pre-commit-hooks. 
Specifically, it is considered a completed sentence, which should end with a dot. 
Since it won't affect the jump from the link, I think we can keep it.